packages/investor-foxy/src/api/api.ts

```diff
@@ -53,14 +53,14 @@ import {
 
 export * from './foxy-types'
 
+// EthereumMainnet, EthereumRinkeby, EthereumRopsten
+type Network = '1' | '3' | '4'
```

Comment:
Reason to not use constant strings over hardcode?

Reply:
There was an issue using the `CHAIN_REFERENCE` constant due to a circular import, or something similar.

It was a battle I chose not to fight last night - I'll take another look today and add an inline comment for clarity if I can't work it out.

Reply:
Circular import strikes me as odd considering the caip package where they are defined should not be importing any "feature" packages in lib. But that sounds good, let me know.

Reply:
Committed what we landed on in https://github.com/shapeshift/lib/pull/656/commits/8226d6f9308f7999c4154eb4c3b36ae6be5c7d39